### PR TITLE
Creating the helm structure for argocd

### PR DESCRIPTION
**Pull request by RJaliliSTFC**
Added template yaml files for each application

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (DavidFair) [inline Chart.yaml:24]: We should drop this, since we're not using it
But I'm not sure if helm requires it?

File: templates/thanos.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+kind: Application
+metadata:
+  name: thanos-{{ $.Values.global.env }}
+  namespace: argocd
+spec:
+  project: {{ $.Values.global.spec.project }}
+  sources:
+  - repoURL: 'https://github.com/bitnami/charts.git'
+    path: bitnami/thanos
+    targetRevision: HEAD
```
Comment (DavidFair) [inline templates/thanos.yaml:11]: We should have a specific SHA / tag

File: values/harbor-values.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+expose:
+    type: loadBalancer
+    tls:
+        enabled: false
+loadBalancer:
+    IP: "130.246.214.222"
```
Comment (DavidFair) [inline values/harbor-values.yaml:6]: We should drop this IP for the moment, as it's not specific to anything

**Review by DavidFair — CHANGES_REQUESTED** [review]

File: Chart.yaml (hunk 1 above)
Comment (RJaliliSTFC) [reply Chart.yaml:24]: This is needed

File: templates/thanos.yaml (hunk 2 above)
Comment (RJaliliSTFC) [reply templates/thanos.yaml]: resolved in 4742c4ee91df7b911bfcf411c6d336050efbc76d

File: values/harbor-values.yaml (hunk 3 above)
Comment (RJaliliSTFC) [reply values/harbor-values.yaml]: resolved in 4742c4ee91df7b911bfcf411c6d336050efbc76d

**Review by DavidFair — APPROVED** [review]

**Pull request merged**